tests/helpers/deployments/test_deployments.py

```diff
@@ -62,7 +62,7 @@
 description: Description
 aea_version: ">=1.0.0, <2.0.0"
 license: Apache-2.0
-agent: "valory/oracle:0.1.0"
+agent: valory/oracle:0.1.0:bafybeihurloujnbugvvrv5xegyrdnsgl6z6at5xilq7f5ynjdekijapt6q
```

Comment:
This hash is not a current version...what is it pointing to? It should always match the version in packages/hashes.csv

Reply:
This is just a dummy config, we just use it to test if the service loader and deployment generators work or not.

Reply:
Ok